common/lib/xmodule/xmodule/vertical_module.py

```diff
@@ -1,23 +1,34 @@
+"""
+VerticalBlock - a pure XBlock.
+"""
```

Comment:
Yeah, I think that would be good (we're not supporting anyone who subclassed vertical_module anyway, given the current set of changes).